### PR TITLE
DOC: add U of Toronto

### LINKED ISSUE #31: Physics, Utoronto

### PR DESCRIPTION
**Pull request by chiahaoliu**
closes #31

### CODE REVIEW

File: db/institutions.yml [hunk 1]
```diff
@@ -262,4 +262,11 @@ nyu:
   name: New York University
   state: NY
   zip: 10003
-
+utoronto:
+  aka: [UofT, uoft]
+  city: Toronto
+  country: Canada
+  departments: {physics: Department of Physics, chemistry: Department of Chemistry, matsci: Department of Materials Science and Engineering}
```
Comment (sbillinge) [inline db/institutions.yml]: this should be a dict of dicts.  Please see the other examples.

File: db/institutions.yml [hunk 2]
```diff
@@ -262,4 +262,11 @@ nyu:
...
+  name: University of Toronto
+  state: ''
```
Comment (sbillinge) [inline db/institutions.yml]: ontario

Comment (sbillinge) [inline db/institutions.yml]: please also add a street address if you can and a zip if thre is one.

**Comment by chiahaoliu** [comment]
@sbillinge Thanks for the review. I just pushed the fixes.

One thing I found is the zip is `M5S 3H7` which is a mix of string and int. Will this break the db?

**Comment by sbillinge** [comment]
M5S 3H7 is just a string, so no problem.

**Pull request merged**